openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -37,32 +37,32 @@ def _make_profile_image_name(username):
     return hashlib.md5(settings.PROFILE_IMAGE_SECRET_KEY + username).hexdigest()
 
 
-def _get_profile_image_filename(name, size):
+def _get_profile_image_filename(name, size, version):
     """
     Returns the full filename for a profile image, given the name and size.
     """
-    return '{name}_{size}.jpg'.format(name=name, size=size)
+    return '{name}_{size}.{version}.jpg'.format(name=name, size=size, version=version)
 
 
-def _get_profile_image_urls(name):
+def _get_profile_image_urls(name, version):
     """
     Returns a dict containing the urls for a complete set of profile images,
     keyed by "friendly" name (e.g. "full", "large", "medium", "small").
     """
     storage = get_profile_image_storage()
     return {
-        size_display_name: storage.url(_get_profile_image_filename(name, size))
+        size_display_name: storage.url(_get_profile_image_filename(name, size, version))
```

Comment:
One challenge I see is how we are going to clean up old images in the future. Currently, when we clear the user's profile image when they set their year of birth, we don't remove the image. We'd discussed the notion of having a background task that cleans up unwanted images, but I'm not sure we'll know what they are anymore. I was thinking that the filename would be constant in S3, but that the URL we use to look it up would have the version added just for cache busting. If the filename includes a timestamp, though, I don't know that this background process can determine which files to remove.